onmt/translate/Translator.lua

```diff
@@ -18,7 +18,9 @@ local options = {
   {'-pre_filter_factor', 1, [[Optional, set this only if filter is being used. Before
                             applying filters, hypotheses with top `beamSize * preFilterFactor`
                             scores will be considered. If the returned hypotheses voilate filters,
-                            then set this to a larger value to consider more.]]}
+                            then set this to a larger value to consider more.]]},
+  {'-print_input_embeddings', false, [[Instead of generating target tokens conditional on
```

Comment:
I find this option name ambiguous mostly because embeddings usually refer to word embeddings. What about `dump_input_encoding`?